sql/telemetry/clients_histogram_aggregates_v1/query.sql

```diff
@@ -126,8 +127,7 @@ normalized_histograms AS
       key,
       agg_type,
       latest_version,
-      udf_normalized_sum(
-        udf_aggregate_map_sum(ARRAY_AGG(STRUCT<key_value ARRAY<STRUCT <key STRING, value INT64>>>(value)))) AS aggregates
+      udf_normalized_sum(udf_map_sum(ARRAY_CONCAT_AGG(value))) AS aggregates
```

Comment:
r? @emtwo on modifying `clients_histogram_aggregates` to handle the updated `udf_map_sum` function. This change shouldn't impact the input or output of this query.